portals/devportal/source/src/app/components/Apis/Details/GraphQLConsole/GraphQLUI.jsx

```diff
@@ -80,21 +80,34 @@ export default function GraphQLUI(props) {
         setIsExplorerOpen(newExplorerIsOpen);
     };
 
+    const getURLs = () => {
```

Comment:
return value should be a value

Reply:
Fixed with https://github.com/wso2/apim-apps/pull/130/commits/d1df74c14478f7df3a3afd0f7bca66d3511cba20